Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Builder
+FROM golang:alpine as builder
+
+WORKDIR /build
+COPY . /build
```

Comment:
There is no need to specify /build after you have switched with `WORKDIR /build`. `COPY . .` will suffice.

To enable caching of dependencies, I'd like to see this section before `COPY . .`:

```
COPY go.mod go.sum .
go mod download
```